### PR TITLE
Colorful ActionBar, tiny bugfixes and equal errors

### PR DESCRIPTION
**Pull request by gediminasel**
Peržiūrėkit, pratestuokit ir pamerge'inkit

### CODE REVIEW

File: app/src/main/java/com/nmakademija/nmaakademija/MainActivity.java [hunk 1]
```diff
@@ -27,28 +28,34 @@ protected void onCreate(Bundle savedInstanceState) {
...
                 break;
             case 2:
                 fragment = TimeUntilSessionFragment.getInstance();
+                bar.setBackgroundDrawable(new ColorDrawable(getResources().getColor(R.color.bottomNavigationItem3)));
+                bar.setSubtitle(getResources().getString(R.string.timer));
+                break;
+            case 3:
+                fragment = ScheduleFragment.getInstance();
+                bar.setBackgroundDrawable(new ColorDrawable(getResources().getColor(R.color.bottomNavigationItem4)));
+                bar.setSubtitle(getResources().getString(R.string.schedule));
```
Comment (vycius) [inline app/src/main/java/com/nmakademija/nmaakademija/MainActivity.java]: Create two variables before switch: `color` and `subTitle`. Set these inside switch. After switch call:

``` java
bar.setBackgroundDrawable(new ColorDrawable(getResources().getColor(color)));
bar.setSubtitle(getResources().getString(subTitle));
```

File: app/src/main/java/com/nmakademija/nmaakademija/MainActivity.java [hunk 2]
```diff
@@ -58,7 +65,7 @@ void setCurrentFragment(int position) {
 
     @Override
     public void onMenuItemSelect(@IdRes final int itemId, final int position) {
-        setCurrentFragment(position);
+        setCurrentFragment(position, getSupportActionBar());
```
Comment (vycius) [inline app/src/main/java/com/nmakademija/nmaakademija/MainActivity.java]: There is no need to pass `ActionBar`. Get it inside`setCurrentFragment`.

File: app/src/main/res/values/colors.xml [hunk 3]
```diff
@@ -4,4 +4,8 @@
     <color name="colorPrimaryDark">#303F9F</color>
     <color name="colorAccent">#FF4081</color>
     <color name="blackText">#000000</color>
+    <color name="bottomNavigationItem1">#3F51B5</color>
```
Comment (vycius) [inline app/src/main/res/values/colors.xml]: We need clearer names for these bottom navigation items. E.g `bottomNavigationNewsTab`, `bottomNavigationScheduleTab`

File: app/src/main/AndroidManifest.xml [hunk 4]
```diff
@@ -9,7 +9,9 @@
         android:supportsRtl="true"
         android:theme="@style/AppTheme">
 
-        <activity android:name=".MainActivity">
+        <activity
+            android:name=".MainActivity"
+            android:configChanges="orientation|screenSize">
```
Comment (vycius) [inline app/src/main/AndroidManifest.xml:14]: Why ?

**Comment by vycius** [comment]
👍 LGTM. Mentioned small code style improvements. Feel free to merge with master when you are ready.

File: app/src/main/AndroidManifest.xml (hunk 4 above)
Comment (gediminasel) [reply app/src/main/AndroidManifest.xml:14]: If you rotate device fragment changes to default(news) and bottom navigation doesn't change. This line fixes it(left last opened fragment)

**Pull request merged**